util/io/src/worker.rs

```diff
@@ -68,11 +66,8 @@ impl Worker {
 		};
 		worker.thread = Some(thread::Builder::new().name(format!("IO Worker #{}", index)).spawn(
 			move || {
-				panic_handler.catch_panic(move || {
-					Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
-				}).unwrap()
-			})
-			.expect("Error creating worker thread"));
+				Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
```

Comment:
It won't panic? Or is there some other way parity will be closed in case of panic in IO Worker thread?